http_service/bugbug_http/app.py

```diff
@@ -230,6 +231,29 @@ def create_bug_classification_jobs(
     )
 
 
+def create_broken_site_report_classification_jobs(
+    model_name: str, reports: list[dict]
+) -> tuple[JobInfo, str, int]:
+    """Create job_id and redis connection"""
+    job_id = get_job_id()
+
+    # Set the mapping before queuing to avoid some race conditions
+    job_id_mapping = {}
+    for report in reports:
+        key = JobInfo(
+            classify_broken_site_report, model_name, report["uuid"]
+        ).mapping_key
+        job_id_mapping[key] = job_id
+
+    redis_conn.mset(job_id_mapping)
+
+    return (
+        JobInfo(classify_broken_site_report, model_name, reports),
+        job_id,
+        BUGZILLA_JOB_TIMEOUT,
```

Comment:
nit: It looks a bit confusing to use `BUGZILLA_JOB_TIMEOUT` here since there is nothing related to Bugzilla in this job. We could drop it here, and then it will use the default timeout. Alternatively, we could use `JOB_TIMEOUT`, which is the same as the default timeout.